samples/msal-node-samples/refresh-token/adalApp.js

```diff
@@ -0,0 +1,99 @@
+// Import dependencies
+var express = require('express');
+var session = require('express-session');
+var crypto = require('crypto');
+var adal = require('adal-node');
+
+var config = require('./config/customConfig.json');
+
+var DiskCache = require('./adalCustomCache');
+var diskCache = new DiskCache(config.adalCacheLocation);
+
+// Authentication parameters
+var clientId = config.clientId;
+var clientSecret = config.clientSecret;
+var tenant = config.tenantInfo;
+var authorityUrl = config.authority + '/' + tenant;
```

Comment:
Should you use a template literal here (and everywhere else in this review where applicable)?

var authorityUrl = `${config.authority}/${tenant}`;

I personally think it makes the code look much nicer. Up to you though.

Reply:
I love template literals too! It's just in the ADAL sample I'm trying to stick to pre-ES6

Reply:
I forgot that we discussed that already. Sorry!